sonar/modules/documents/views.py

```diff
@@ -330,6 +320,27 @@ def contribution_text(contribution):
     return ' '.join(data)
 
 
+@blueprint.app_template_filter()
+def meeting_text(contribution):
+    """Display meeting format.
+
+    :param contribution: Dict representing the contribution.
+    :returns: Formatted text.
+    """
+    meeting = []
+    if contribution['agent'].get('number'):
```

Comment:
You can use a loop over the subfields or better something like:
```python
contrib = contribution['agent']
return ' : '.join([value for (key, value) in contrib.items() if key in ['number', 'date', 'place']])
```
You can also use the python filter function if you want.